modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -0,0 +1,353 @@
+= Mule Upgrade Tool
+
+*Latest version: 1.0.0*
+
+include::release-notes::partial$mule-upgrade-tool/mule-upgrade-tool.adoc[tag=intro]
+
+The Mule upgrade tool runs in any environment that is capable of running a standalone Mule instance. This tool can update only standalone Mule instances that are running any Mule 4 version.
+//You can upgrade local Mule instances to the latest available version or to a particular Mule distribution that you have previously downloaded and unpacked.
+You can also use the Mule upgrade tool to update Mule instances located in different folders.
```

Comment:
Remove the term "also". Proposed text here:

You can use the Mule upgrade tool to update Mule instances located in different folders.